packages/odyssey-react-mui/tsconfig.production.json

```diff
@@ -2,9 +2,11 @@
   "extends": "@okta/odyssey-typescript/tsconfig.react.production.json",
   "compilerOptions": {
     "composite": true,
+    "emitDeclarationOnly": true,
     "jsx": "react-jsx",
     "jsxImportSource": "@emotion/react",
-    "outDir": "dist"
+    "rootDir": "src",
```

Comment:
We can't do `rootDir` as `src/` because it will cause issues with files at the root of the package that need to be seen by the tsconfig.

For Prod, we can do this, but the issue becomes we run into issues where the exports for dev and prod mode are different.

So it's easier to leave it as `src` even though it's ugly in the `dist/` folder.